conf/vertebrates/mlss_conf.xml

```diff
@@ -34,98 +34,275 @@
+    <one_vs_all method="LASTZ_NET" ref_genome="ornithorhynchus_anatinus" against="Prototheria"/>
 
-    <!-- Early vertebrates and other chordates -->
-    <one_vs_all method="LASTZ_NET" ref_genome="ciona_intestinalis" against="Ciona" ref_amongst="Cyclostomata"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="petromyzon_marinus" against="Cyclostomata" ref_amongst="Ciona"/>
+    <!-- Sauropsids -->
+    <one_vs_all method="LASTZ_NET" ref_genome="gallus_gallus">
+      <species_set>
+        <taxonomic_group taxon_name="Sauropsida"/>
+        <!-- Remove all the taxa that have a better reference than chicken -->
+        <taxonomic_group taxon_name="Iguania" exclude="1"/>
+      </species_set>
+    </one_vs_all>
+
+    <one_vs_all method="LASTZ_NET" ref_genome="anolis_carolinensis" against="Iguania"/>
 
-    <!-- These genomes are currently the only ones in their clades, so no alignment within the "against"/"species_set" -->
-    <!-- However they will be aligned against the other reference genomes in "ref_amongst" -->
-    <one_vs_all method="LASTZ_NET" ref_genome="equus_caballus" against="Perissodactyla" ref_amongst="Laurasiatheria"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="sus_scrofa" against="Suina" ref_amongst="Laurasiatheria"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="anolis_carolinensis" against="Lepidosauria" ref_amongst="Sauropsida"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="pelodiscus_sinensis" against="Testudines" ref_amongst="Sauropsida"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="xenopus_tropicalis" against="Amphibia" ref_amongst="Euteleostomi"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="ornithorhynchus_anatinus" against="Prototheria" ref_amongst="Euteleostomi"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="latimeria_chalumnae" ref_amongst="Euteleostomi">
+    <!-- Fishes -->
+    <one_vs_all method="LASTZ_NET" ref_genome="oryzias_latipes">
       <species_set>
-        <!--Coelacanths and lungfishes -->
-        <taxonomic_group taxon_name="Sarcopterygii"/>
+        <!-- Include urochordates, cephalochordates, cyclostomes, cartilaginous fishes, ray-finned fishes, coelacanths and lungfishes -->
+        <taxonomic_group taxon_name="Chordata"/>
         <taxonomic_group taxon_name="Tetrapoda" exclude="1"/>
       </species_set>
     </one_vs_all>
+
+    <!-- Early vertebrates and other chordates -->
+    <one_vs_all method="LASTZ_NET" ref_genome="xenopus_tropicalis" against="Amphibia"/>
+    <one_vs_all method="LASTZ_NET" ref_genome="ciona_intestinalis" against="Ciona" ref_amongst="Cyclostomata"/>
+    <one_vs_all method="LASTZ_NET" ref_genome="petromyzon_marinus" against="Cyclostomata" ref_amongst="Ciona"/>
+
   </pairwise_alignments>
 
   <multiple_alignments>
+
     <!-- Primates -->
-    <multiple_alignment method="EPO+EPO_EXTENDED">
+    <multiple_alignment method="EPO">
+      <species_set name="primates">
+        <genome id="60"/>
+        <genome id="150"/>
```

Comment:
If you make this one `<genome name="homo_sapiens"/>` the script will be able to include it in the new mlss_id when it gets an assembly update (indeed we would like to recompute EPO). Same for all reference genomes (human, mouse, others ? I don't even know the correct list...) in other EPO alignments too (but not Pecan, right ?).

This way, we wouldn't need to manually update this file throughout the scale down